exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -194,10 +194,12 @@ drill.exec: {
   udf: {
     retry-attempts: 5,
     directory: {
-      base: ${drill.home}"/"${drill.exec.zk.root}"/udf",
-      staging: ${drill.exec.udf.directory.base}"/staging",
-      registry: ${drill.exec.udf.directory.base}"/registry",
-      tmp: ${drill.exec.udf.directory.base}"/tmp"
+      base: ${drill.exec.zk.root}"/udf",
+      local: ${drill.exec.udf.directory.base}"/local",
```

Comment:
How is this used? The path given by local seems relative:

drill/udf

Maybe include a simple comment to describe each config setting.

Otherwise, I like how this has come together!